cli/internal/helm/upgrade.go

```diff
@@ -174,11 +172,11 @@ func (c *UpgradeClient) Upgrade(ctx context.Context, config *config.Config, idFi
 	// Backup CRDs and CRs if we are upgrading anything.
 	if len(upgradeReleases) != 0 {
 		c.log.Debugf("Creating backup of CRDs and CRs")
-		crds, err := c.backupCRDs(ctx, upgradeID)
+		crds, err := c.backupCRDs(ctx, upgradeDir)
```

Comment:
The backup file structure should not have changed from this PR.
`upgradeDir` is simply `upgradeWorkspace/upgradeID`.
We previously kept the workspace as a variable of the helm upgrade, which is not necessary.
Since this PR removed the need to keep the base upgradeWorkspace directory and upgradeID separate, I refactored the Helm code to also not need to know the ID.

Reply:
Created a ticket to add documentation for backups.

Reply:
i see the paths are the same, lgtm